### PR TITLE
Cherry pick initscript fixes

### PR DESCRIPTION
**Pull request by ehelms**
For plugin updates, please indicate which repos this should be built into:

* [x] Nightly
* [ ] 1.18
* [ ] 1.17
* [ ] 1.16

See Foreman's [plugin maintainer documentation](https://projects.theforeman.org/projects/foreman/wiki/How_to_Create_a_Plugin#Release-strategies) for more information.

---

### CODE REVIEW

File: packages/foreman/foreman/foreman.spec [hunk 1]
```diff
@@ -1274,6 +1271,9 @@ exit 0
 %systemd_postun_with_restart %{name}.service
 
 %changelog
+* Mon Jul 16 2018 Eric D. Helms <ericdhelms@gmail.com> - 1.18.0-2
+- rebuilt
```
Comment (ekohl) [inline packages/foreman/foreman/foreman.spec]: I think we could be a bit more descriptive here :)

File: packages/katello/rubygem-qpid_messaging/rubygem-qpid_messaging.spec [hunk 2]
```diff
@@ -70,6 +71,9 @@ rm -rf %{buildroot}%{gem_instdir}/ext
 %license %{gem_instdir}/LICENSE
 
 %changelog
+* Fri Jul 13 2018 Eric D. Helms <ericdhelms@gmail.com> - 1.36.0-2
+- rebuilt
```
Comment (ekohl) [inline packages/katello/rubygem-qpid_messaging/rubygem-qpid_messaging.spec]: And here :)

**Comment by ehelms** [comment]
[test rpm]

File: packages/katello/rubygem-qpid_messaging/rubygem-qpid_messaging.spec (hunk 2 above)
Comment (ehelms) [reply packages/katello/rubygem-qpid_messaging/rubygem-qpid_messaging.spec]: This was a cherry pick so I'd rather not change it.

File: packages/foreman/foreman/foreman.spec (hunk 1 above)
Comment (ehelms) [reply packages/foreman/foreman/foreman.spec]: I can update this.

File: packages/katello/rubygem-qpid_messaging/rubygem-qpid_messaging.spec (hunk 2 above)
Comment (ekohl) [reply packages/katello/rubygem-qpid_messaging/rubygem-qpid_messaging.spec]: Fair enough

**Pull request merged**